### ISSUE TITLE
Power Commands "Copy/Paste Class" does not show up in context menu (VS2017)

### ISSUE THREAD

**Issue by KerbalSpike** [Microsoft/VS-PPT#133]
When I right click on any .cs file in my solution, I do not get the "Copy/Paste Class commands", only "Copy Path" shows up.

There seem to be other inconsistencies as well:
"Copy Reference" for references is available.
"Extract Constant" for text in the editor is not available.

I've got 2 instances of VS2017 installed: Community and Enterprise. I have uninstalled and reinstalled PPT2017 for both and the issue persists. Resetting PPT2017 does not help and I also tried if not using elevated privileges makes any difference - it doesn't.

Power Commands are installed and enabled and all commands are enabled in the options menu (also tried disabling/enabling individual commands - again, makes no difference).

I know that I have used that feature before, but I unfortunately I do not know when/why it disappeared.

**Comment by justcla** [comment]
In my testing on VS2017 I can see "Copy Class" when I right-click on a class in Solution Explorer.
And I see "Paste Class" when I right-click a project node (under Power Commands menu) or when I right-click a folder inside a project.

I am not sure why this command is not showing for you. Do you see the Power Commands menu in the context menu of a Project?

Please raise separate issues for the other inconsistencies. Thanks.

**Comment by KerbalSpike** [comment]
Yes, I do see the Power Commands menu for Projects and Solutions.
I have attached 2 screenshots, one with the context menu of a project and one showing the conext menu of a .cs file containing just a class (inside a namespace).
![context_menu_classfile](https://cloud.githubusercontent.com/assets/18031240/25988865/0b9200c0-36fa-11e7-9986-1afd099df930.jpg)
![context_menu_project](https://cloud.githubusercontent.com/assets/18031240/25988864/0b8f0c4e-36fa-11e7-994c-387b7fd0cc40.jpg)

Are there any configuration and/or log files I can provide to help solve this issue?

**Comment by gernotpokorny** [comment]
same issue here. vs2017 v15.2 "Copy Class" is not available
